### PR TITLE
feat: Support usePreinstalledWDA on simulators

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Related to https://github.com/appium/WebDriverAgent/pull/866

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa Would you have time to test this feature locally?

**Review by KazuCocoa — APPROVED** [review]

**Review by KazuCocoa — CHANGES_REQUESTED** [review]
mm, wait. seeing a bit weird behavior (just to drop the approval)

File: lib/driver.js [hunk 1]
```diff
@@ -818,38 +807,10 @@ class XCUITestDriver extends BaseDriver {
...
-          // Over Xcode 10 will often try to access the app from its staging
-          // directory before fully moving it there, and fail. Retrying once
-          // immediately helps
-          this.cachedWdaStatus = await retry(
-            2,
-            this.wda.launch.bind(this.wda),
-            sessionId,
-            realDevice,
-          );
+          this.cachedWdaStatus = this.wda.launch(sessionId);
```
Comment (KazuCocoa) [inline lib/driver.js:813]:
```suggestion
          this.cachedWdaStatus = await this.wda.launch(sessionId);
```
?

Comment (mykola-mokhnach) [reply lib/driver.js:813]: good catch, thanks

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**